sktime/forecasting/pytorchforecasting.py

```diff
@@ -378,3 +523,529 @@ def get_test_params(cls, parameter_set="default"):
             ]
 
         return params
+
+
+class PytorchForecastingDeepAR(_PytorchForecastingAdapter):
```

Comment:
Is the DeepAR forecaster already part of a different PR? (Might also be the case that I am confused :) )

Reply:
Yes, DeepAR and NHiTS are part of #6551, this branch is inherited from that.